requirements-test.txt

```diff
@@ -1,5 +1,6 @@
-pytest>=7.0.1
-moto>=3.0.7
 codecov>=2.1.12
+daal4py==2023.0.1
```

Comment:
These are already part of requirements-optional.txt. You should install it either manually in your local environment (my preferred approach) or install the entire requirements-optional.txt file.

Also note that MLFlow is another optional library that causes test discovery to fail. This is also covered in requirements-optional.txt hence, it is not needed here.

Reply:
just checked. daal4py is **not** in the `requirements-optional.txt` file. Or is it a dependency of another package?

Reply:
I think it is installed by this optional dependency

```
scikit-learn-intelex
```